src/app/pages/unlock.tsx

```diff
@@ -32,10 +33,7 @@ export function Unlock() {
     }
   }, [hasApprovedNewBrand, navigate]);
 
```

Comment:
Do we always want to go home now after this? Or back to the previous location?

Reply:
to `Home`, otherwise leads to bugs. discussed this in slack